compiler/preprocess.go

```diff
@@ -91,13 +92,7 @@ func (p *Preprocess) checkCreateLabelStmt(stmt *ast.CreateLabelStmt) {
 	}
 
 	if !stmt.IfNotExists {
-		graphName := p.sc.CurrentGraph()
-		if graphName == "" {
-			p.err = ErrGraphNotChosen
-			return
-		}
-
-		graph := p.sc.Catalog().Graph(graphName)
+		graph := p.sc.CurrentGraph()
```

Comment:
You have changed the semantic here. An `ErrGraphNotChosen` are expected if users didn't choose graph via `USE <graphName>`